samples/msgext-unfurling-ac-loop-components/csharp/Resources/adaptiveCard.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "type": "AdaptiveCard",
+  "version": "1.5",
+  "metadata": {
+    "webUrl": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main"
+  },
+  "body": [
+    {
+      "type": "TextBlock",
+      "text": "Adaptive Card-based Loop component",
+      "size": "Large",
+      "weight": "Bolder"
+    },
+    {
+      "type": "Container",
+      "items": [
+        {
+          "type": "TextBlock",
+          "text": "These samples are designed to help understand Microsoft Teams platform capabilities and scenarios(Bots,Tabs,Message extensions,Meeting extensions,Personal apps,Webhooks and connectors)",
```

Comment:
Give one space after every comma

Reply:
Still it is not resolved.

(Bots, Tabs, Message extensions, Meeting extensions, Personal apps, Webhooks and connectors)